api/v1/zz_generated.deepcopy.go

```diff
@@ -261,6 +261,11 @@ func (in *Driver) DeepCopyInto(out *Driver) {
 			(*in)[i].DeepCopyInto(&(*out)[i])
 		}
 	}
+	if in.ForceRemoveDriver != nil {
```

Comment:
is this modified manually or regenerated?

Reply:
It got regenerate. I haven't added this.